sdk/identity/azure-identity/tests/managed-identity-live/service-fabric/service_fabric.md

```diff
@@ -0,0 +1,190 @@
+
+The `sfmitestsystem` and `sfmitestuser` directories contain mock applications that use Azure.Identity's `ServiceFabricCredential` to request and verify Key Vault access tokens. The former application uses a system-assigned managed identity to do so, and the latter application uses a user-assigned managed identity.
+
+> **Note:** The code run by the applications comes from the `Dockerfile` in each, so adapting this sample to another language can be done by editing the Docker image each application uses. No other configuration changes are necessary if using Linux containers. To use Windows containers, you would need to edit the cluster configuration to run on Windows virtual machines (an example of a Windows configuration can be found [here](https://github.com/Azure/azure-quickstart-templates/tree/master/service-fabric-secure-cluster-5-node-1-nodetype)).
+
+The `arm-templates` directory contains Azure resource templates for creating a Service Fabric cluster to host these applications as well as the application templates.
+
+### Environment requirements
+
+> **Note:** All Azure resources used in the sample should be in the same region & resource group. This includes a managed identity, Key Vault, Service Fabric cluster, Azure Container Registry, and storage account.
+
+- This sample requires access to an Azure subscription and required privileges to create resources.
+- [Powershell and the Az library are needed to run the deployments in the sample.](https://docs.microsoft.com/powershell/azure/install-az-ps)
+- [Azure CLI is used to deploy some resources.](https://docs.microsoft.com/cli/azure/install-azure-cli?view=azure-cli-latest)
+- Docker is needed to build and push the sample containerized service. Docker should be using Linux containers for building the application images that are provided.
+
+### Clone this repository
+
+From a command prompt window, run
+```
+git clone https://github.com/Azure/azure-sdk-for-python --single-branch --branch master --depth 1
+cd azure-sdk-for-python/sdk/identity/azure-identity/tests/managed-identity-live/service-fabric
+```
+
+### Sections
+- [Resource setup](#set-up-resources)
+- [Application deployment](#set-up-and-deploy-the-applications)
+- [Test validation](#run-the-tests)
+
+## Set Up Resources
+
+You can skip to [Set Up and Deploy the Applications](#set-up-and-deploy-the-applications) if you have an existing Service Fabric cluster, key vault, storage account, container registry, and managed identity named "AdminUser".
+
+### Create a resource group
+
+From a command prompt window, run
+```
+az login
+az group create -n $RESOURCE_GROUP --location $LOCATION --subscription $SUBSCRIPTION_NAME
+```
+
+### Create a user-assigned managed identity
+
+From your command prompt window, run
+```
+az identity create -g $RESOURCE_GROUP -n AdminUser
+```
+
+Make note of the identity's principal ID, to use in the next step. Also make note of its client ID for a later step.
```

Comment:
What if I have a resource group already? How do I get it?

Reply:
Good point! I added a note of how to fetch the IDs of an existing identity